x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -34,10 +39,61 @@ export interface Props {
 
 interface State {
   isPopoverOpen: boolean;
+  isLayerEditable: boolean;
```

Comment:
Can you rename isLayerEditable to supportsFeatureEditing to avoid confusion about layer settings editing and feature editing. I know this sounds nit-picky but its really important for maintainability to clear understand what these variables mean and not have multiple meanings for a variable.